cmd/createtree/pem_test.go

```diff
@@ -0,0 +1,106 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/golang/protobuf/ptypes"
+	"github.com/google/trillian/crypto/keys"
+	"github.com/google/trillian/crypto/keyspb"
+)
+
+func TestWithPEMKeyFile(t *testing.T) {
+	pemPath, pemPassword := "../../testdata/log-rpc-server.privkey.pem", "towel"
+
+	wantTree := *defaultTree
+	keyProto, err := ptypes.MarshalAny(&keyspb.PEMKeyFile{
+		Path:     pemPath,
+		Password: pemPassword,
+	})
+	if err != nil {
+		t.Fatalf("MarshalAny(PEMKeyFile): %v", err)
+	}
+	wantTree.PrivateKey = keyProto
+
+	runTest(t, []*testCase{
+		{
+			desc: "empty pemKeyPath",
```

Comment:
Since this is tested in an integration-like style, where we exercise it via createtree, I'd be tempted to use real handlers at main_test.go and move those test cases there. Or, possibly, just test the handlers directly here. If you take the comment on package separation, it'll force you down one of those roads regardless (which why I like it).